src/mono/msbuild/android/build/AndroidBuild.targets

```
@@ -47,6 +46,9 @@
     </PropertyGroup>
 
     <ItemGroup Condition="'$(_IsLibraryMode)' == 'true'">
+      <_CommonLinkerArgs Include="-l:libz.so;-l:liblog.so;-l:libc.so;-l:libm.so;" />
```

nit: I think it'd be more readable like this:
```suggestion
      <_CommonLinkerArgs Include="-l:libz.so" />
      <_CommonLinkerArgs Include="-l:liblog.so" />
      <_CommonLinkerArgs Include="-l:libc.so" />
      <_CommonLinkerArgs Include="-l:libm.so" />
```